### PR TITLE
Move to mustermann (not 19)

### LINKED ISSUE #2166: upgrade mustermann to latest version
#### Do you want to request a *feature* or report a *bug*?

An issue, mustermann19 has the same library name as mustermann. So let's say that we have both grape and padrino in one application then they will complain.

#### What is the current behavior?

Currently Padrino requires mustermann19

#### What is the expected behavior?

It shouldn't complain that it has the same name.

#### Which versions of Ruby, Padrino, Sinatra, Rack, OS are you using? Did this work in previous versions?

Issue is in all current versions of Padrino, Sinatra, Grape.
Ruby v 2.4.3

#### Possible Solutions

Either move lib/mustermann in mustermann19 to lib/mustermann19 or move to mustermann 1.0.1

### PR DESCRIPTION
**Pull request by jnylen**
This is one of the solutions for #2166.
The other being moving the lib/mustermann to lib/mustermann19

Closes #2166

### CODE REVIEW

**Comment by jnylen** [comment]
Some of these builds will fail due to rack version being incompatible.

**Comment by jnylen** [comment]
Not working yet:
```
  ERROR -  NoMethodError - undefined method `+' for #<Mustermann::Regular:"\\/batches(?![^\\/])">:
 /vagrant/vendor/bundle/ruby/2.4.0/gems/sinatra-contrib-2.0.0/lib/sinatra/namespace.rb:332:in `prefixed_path'
bundler: failed to load command: padrino (/vagrant/vendor/bundle/ruby/2.4.0/bin/padrino)
NoMethodError: undefined method `+' for #<Mustermann::Regular:"\\/batches(?![^\\/])">
```

File: .travis.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ branches:
   only:
     - master
     - 0.12
+    - move-to-mustermann
```
Comment (namusyaka) [inline .travis.yml]: This line isn't needed, please remove it.

**Comment by namusyaka** [comment]
I don't see any reason why we should use mustermann19 now.
Thanks for bringing this up.

**Comment by jnylen** [comment]
@namusyaka, fixed.
But jfyi, I haven't fixed the first error as I don't really use ruby anymore.

**Comment by namusyaka** [comment]
We need to get a consensus about the ruby versions we support. #2168 
In my opinion, this Pull Request can be used as it is, but please wait a while until we get a consensus.
Thanks for the patch @jnylen

**Comment by ujifgc** [comment]
Closing in favor of #2169

**Pull request closed without merging**